packages/text-document-utils/test/lineOffsets.test.ts

```diff
@@ -0,0 +1,43 @@
+import { getLineOffsets } from '../src/lineOffsets';
```

Comment:
File name does not match the naming convention. Please use kebab-case.

Reply:
done.